packages/blade/src/components/Box/BaseBox/types/index.ts

```diff
@@ -0,0 +1,3 @@
+export type { BaseBoxProps } from './props.types';
+export type { ArrayOfMaxLength4, SpacingValueType } from './spacing.types';
+export type { MakeValueResponsive } from './responsive.types';
```

Comment:
why `.types` naming convention in files? other places we have things like `spacingTypes` `responsiveTypes`

Reply:
no strong opinion. didn't notice how we were doing in other files. changed it to camelCase.